docs/deep-dives/mobile/nimbus-cli/working-with-files.mdx

```diff
@@ -0,0 +1,58 @@
+---
+id: nimbus-cli-working-with-files
+title: Working with Files
+slug: /nimbus-cli/working-with-files
+---
+
+## `fetch`
+
+You can download from a server either the current list of all experiments:
+
+```sh
+nimbus-cli --app fenix --channel developer fetch archived.json
+```
+
+Use `preview`, `stage` or `stage/preview` to specify a Remote Settings instance.
```

Comment:
What does `stage/preview` do? is that different from specifying just one of the two?

Reply:
There's a preview instance on stage too